src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -531,8 +531,9 @@ private function getPropertySchema(PropertyMetadata $propertyMetadata, \ArrayObj
         }
 
         $valueSchema = $this->getType($builtinType, $isCollection, $className, $propertyMetadata->isReadableLink(), $definitions, $serializerContext);
+        $attributeSchema = $propertyMetadata->getAttributes();
 
-        return new \ArrayObject((array) $propertySchema + $valueSchema);
+        return new \ArrayObject((array) $propertySchema + $valueSchema + ($attributeSchema['swagger_context'] ?? []));
```

Comment:
`$attributeSchema` is useless. `($propertyMetadata->getAttributes()['swagger_context'] ?? [])` should do the trick.